src/Widgets/FormSelectWidget/FormSelectWidgetEditingConfig.ts

```diff
@@ -54,3 +63,37 @@ Scrivito.provideEditingConfig("FormSelectWidget", {
     customFieldNameValidation,
   ],
 });
+
+function getProps(widget: Scrivito.Obj): any[] {
```

Comment:
Are those component `props` or `properties` of widget? 

Maybe there is a type for this (`any[]`) in Scrivito?

Reply:
widget props but somehow it recognise it only as `Obj` and not as `Widget` . in another file it is "Widget" even though the code is almost the same.
The` any[] `was the fastest way. I can try to create a type for it.

Reply:
How those can be widget props as we are in the configuration file?

Reply:
Sorry, i meant those props are the editing properties  to be shown in the widget properties and they take the widget as param

Reply:
Sorry, i meant that those props are the editing properties to be shown in the widget properties and they take "widget" as parameter

Reply:
I would call it `getWidgetProperties` or `getProperties` to not confuse it with React components `props`

Reply:
Right, i changed it to `getProperties`